src/tests/unit/components/ResetPassword.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { CompletePasswordReset } from '../../../containers/authentication/CompletePasswordReset';
+import { RequestPasswordReset } from '../../../containers/authentication/RequestPasswordReset';
+
+describe('CompletePasswordReset', () => {
+  let wrapper;
+  it('Complete password reset has a title of Reset Password', () => {
+    wrapper = shallow(<CompletePasswordReset />).dive();
+    expect(wrapper.findWhere(n => n.type() === 'title' && n.contains('Reset Password')));
```

Comment:
This  test would pass even if there was no `Reset Password` test because it has no matcher. In short what do you expect this line `expect(wrapper.findWhere(n => n.type() === 'title' && n.contains('Reset Password')));` to be or to equal??